ConfigurationSystem/ConfigTemplate.cfg

```diff
@@ -47,6 +47,7 @@ Agents
   }
   GOCDB2CSAgent
   {
+    Cycles = 0
```

Comment:
The situation here is similar as with the NetworkAgent - this agent is also meant to work continuously.